pkg/descheduler/strategies/nodeutilization/nodeutilization.go

```diff
@@ -213,7 +213,7 @@ func evictPodsFromSourceNodes(
 	continueEviction continueEvictionCond,
 ) {
 
-	sortNodesByUsage(sourceNodes)
+	sortNodesByUsage(sourceNodes, strategy)
```

Comment:
Instead of passing the strategy name into `sortNodesByUsage`, would you mind changing the signature to `func sortNodesByUsage(nodes []NodeInfo, ascending bool)` and moving `sortNodesByUsage` function invocation from `evictPodsFromSourceNodes`? I.e.
```go
# LowNodeUtilization
sortNodesByUsage(sourceNodes, false)
evictPodsFromSourceNodes(
		ctx,
		sourceNodes,
		lowNodes,
		podEvictor,
		evictable.IsEvictable,
		resourceNames,
		"LowNodeUtilization",
		continueEvictionCond)
```
and
```go
# HighNodeUtilization
sortNodesByUsage(sourceNodes, true)
evictPodsFromSourceNodes(
		ctx,
		sourceNodes,
		highNodes,
		podEvictor,
		evictable.IsEvictable,
		resourceNames,
		"HighNodeUtilization",
		continueEvictionCond)
```